### DISCUSSION TITLE
Default for vmax/zmax (and min) in cooler show

### DISCUSSION THREAD

**Discussion by aakashsur** [open2c/cooler#369] [Q&A] [answered]
https://github.com/open2c/cooler/blob/82b99394724ab152643a0e4f34d9f097c2a6ee80/cooler/cli/show.py#L237

I was wondering what the defaults were, unless I missed it, I didn't seem to find them listed in `show.py`. The command works fine without them and I was creating some figures to mimic what `cooler show` was doing and was curious what the defaults might be.

**Comment by nvictus** [comment, marked as answer]
The defaults are `None`, which means matplotlib will autoscale to the min/max in the initial view. You may sometimes want to threshold vmax in the case of a strongly contaminated main diagonal.
